src/main/resources/org/archive/commons.properties

```diff
@@ -0,0 +1,5 @@
+operator=BnF
+publisher=Bibliotheque nationale de France
+wat.warcinfo.description=Description du format WAT
```

Comment:
This by default will put BnF specific data in all generated WAT files. Should this be handled differently?